### PR TITLE
Provide mobile_alt interface

### PR DESCRIPTION
**Pull request by apastuhov**
https://jira.camptocamp.com/browse/GSGMF-14

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -696,6 +705,7 @@ node_modules/angular/angular.min.js: .build/node_modules.timestamp
 		--app 'Mobile application' apps/mobile/index.html 'The mobile example application for GeoMapFish.' \
 		--app 'Desktop application' apps/desktop/index.html 'The desktop example application for GeoMapFish.' \
 		--app 'Alternate desktop application' apps/desktop_alt/index.html 'An alternate desktop example application for GeoMapFish.' \
+		--app 'Alternate mobile application' apps/mobile_alt/index.html 'An alternate mobile example application for GeoMapFish.' \
```
Comment (sbrunner) [inline Makefile]: Shouldn't to be above desktop alt? to have mobile, desktop, mobile alt, desktop alt...

Comment (apastuhov) [inline Makefile]: Sorry, I did not see your comment.. I've created separated PR for it: https://github.com/camptocamp/ngeo/pull/2788

Comment (sbrunner) [inline Makefile]: :-)

Comment (apastuhov) [inline Makefile]: I see that you already created a PR for it.. so I will close my PR and remove the branch)

**Pull request merged**